yellowbox/utils.py

```diff
@@ -31,3 +32,8 @@ def get_free_port():
         s.bind(('', 0))
         s.setsockopt(SOL_SOCKET, SO_REUSEADDR, 1)
         return s.getsockname()[1]
+
+if platform.system() == "Linux":
+    _docker_host_name = '172.17.0.1'
```

Comment:
also, if you're moving this to utils, you should replace the usage in other places